tests/test_integration_default.py

```diff
@@ -271,6 +271,19 @@ def test_unused_snapshots_warning(stubs):
     assert result.ret == 0
 
 
+def test_mytest(stubs):
+    _, testdir, tests, _ = stubs
+    testdir.makepyfile(test_file="\n\n".join(tests[k] for k in tests if k != "unused"))
+    testdir.makefile(
+        ".ambr", **{"__snapshots__/other_snapfile": ""},
+    )
+    result = testdir.runpytest("-v", "--snapshot-update", "test_file.py")
+    result_stdout = clean_output(result.stdout.str())
+    assert "1 unused snapshot deleted" in result_stdout
+    assert result.ret == 0
+    assert os.path.isfile("__snapshots__/other_snapfile.ambr")
+
```

Comment:
we should add a test for the image extension (e.g. PNG)

Reply:
Agreed.

Reply:
Since the filtering has been hoisted above the extensions, testing each extensions is pointless (discussed offline).